src/constant.h

```diff
@@ -28,573 +31,573 @@
 // constants at your own risk. -JEW-
 
 // message line location
-#define MSG_LINE 0
+constexpr unsigned int MSG_LINE = 0;
 
 // number of messages to save in a buffer
-#define MAX_SAVE_MSG    22   // How many messages to save -CJS-
+constexpr int MAX_SAVE_MSG = 22; // How many messages to save -CJS-
 
 // Dungeon size parameters
-#define MAX_HEIGHT      66   // Multiple of 11; >= 22
-#define MAX_WIDTH      198   // Multiple of 33; >= 66
-#define SCREEN_HEIGHT   22
-#define SCREEN_WIDTH    66
-#define QUART_HEIGHT    (SCREEN_HEIGHT / 4)
-#define QUART_WIDTH     (SCREEN_WIDTH / 4)
+constexpr int  MAX_HEIGHT = 66; // Multiple of  = 11; >=  = 22
```

Comment:
You have double spacing between `int` and `MAX_HEIGHT`. I guess we don't need this - not just here, but throughout the file.